apps/web/components/Proposals.tsx

```diff
@@ -377,118 +402,132 @@ export function Proposals({
-                  onClick={() => setEditView((prev) => !prev)}
-                >
-                  Cancel
-                </Button>
-                <Button
-                  onClick={() => submit()}
-                  isLoading={allocateStatus === "loading"}
-                  disabled={
-                    !getProposalsInputsDifferences(inputs, stakedFilters).length
-                  }
-                  tooltip="Make changes in proposals support first"
-                >
-                  Save changes
-                </Button>
-              </>
-            )}
-          </div>
+          {proposals?.map((proposalData, i) => (
+            <React.Fragment key={proposalData.id + "_" + i}>
+              <ProposalCard
+                proposalData={proposalData}
+                inputData={inputs[i]}
+                stakedFilter={stakedFilters[i]}
+                i={i}
+                isAllocationView={allocationView}
+                tooltipMessage={tooltipMessage}
+                memberActivatedPoints={memberActivatedPoints}
+                memberPoolWeight={memberPoolWeight}
+                executeDisabled={
+                  proposalData.proposalStatus == 4 ||
+                  !isConnected ||
+                  missmatchUrl
+                }
+                strategy={strategy}
+                tokenDecimals={tokenDecimals}
+                alloInfo={alloInfo}
+                triggerRenderProposals={triggerRenderProposals}
+                inputHandler={inputHandler}
+              />
+            </React.Fragment>
+          ))}
         </div>
-        <div>
-          <h4 className="text-2xl">Do you have a great idea?</h4>
-          <div className="flex items-center gap-6">
-            <p>Share it with the community and get support !</p>
-            <FormLink href={createProposalUrl} label="Create Proposal" />
+        {allocationView && (
+          <div className="flex justify-end gap-4">
+            <>
+              <Button
+                btnStyle="outline"
+                color="danger"
+                onClick={() => setAllocationView((prev) => !prev)}
+              >
+                Cancel
+              </Button>
+              <Button
+                onClick={() => submit()}
+                isLoading={allocateStatus === "loading"}
+                disabled={
+                  !getProposalsInputsDifferences(inputs, stakedFilters).length
+                }
+                tooltip="Make changes in proposals support first"
+              >
+                Save changes
+              </Button>
+            </>
           </div>
-        </div>
+        )}
+
+        {!allocationView && (
+          <>
+            <div>
+              <h4>Do you have a great idea?</h4>
+              <div className="flex items-center gap-6">
+                <p>Share it with the community and get support!</p>
+                <FormLink href={createProposalUrl} label="Create Proposal" />
```

Comment:
try to merge dev into your branch, i remember that we deleted FormLink component as it was only a Link wrapping a Button and there was no need for a component te be created only for this.